services/static-webserver/Dockerfile

```
@@ -0,0 +1,30 @@
+FROM joseluisq/static-web-server:1.15.0-alpine as base
+
+LABEL maintainer=neagu@itis.swiss
+
+# simcore-user uid=8004(scu) gid=8004(scu) groups=8004(scu)
+ENV SC_USER_ID=8004 \
+    SC_USER_NAME=scu \
+    SC_BUILD_TARGET=base \
+    SC_BOOT_MODE=default
+
+RUN adduser \
+    --uid ${SC_USER_ID} \
+    --disabled-password \
+    --gecos "" \
+    --shell /bin/sh \
+    --home /home/${SC_USER_NAME} \
+    ${SC_USER_NAME}
+
+# front-end client last (image name is the path to the Dockerfile)
+COPY --from=client/tools/qooxdoo-kit/builder:latest --chown=scu:scu \
+    /project/build-output "/static-content"
+
```

```suggestion
USER ${SC_USER_NAME}
```
this will ensure the user is scu when starting. maybe check that everything works. but at least locally seemed ok.

---

dialling non root user due to https://github.com/joseluisq/static-web-server/issues/42

---

Thanks to some great support form the static-web-server project, the issue with non root users has been solved. The application was also patched for better error reporting.

---

cool, although I tried it on my linux machine before without seeing any issue.

---

It had to do something with how my development machine was setup. Not sure exactly what.